realm/realm-library/src/androidTest/java/io/realm/RealmQueryTests.java

```diff
@@ -99,14 +99,14 @@ private void populateTestRealm(Realm testRealm, int dataSize) {
             allTypes.setColumnBinary(new byte[]{1, 2, 3});
             allTypes.setColumnDate(new Date(DECADE_MILLIS * (i - (dataSize / 2))));
```

Comment:
I agree with that, but this PR is focusing on fixing warnings from `ErrorProne`.
We should have another PR to fix  that.